libs/common/src/errors/amendment-not-created.error.ts

```diff
@@ -0,0 +1,11 @@
+import { HttpStatusCode } from 'axios';
+import { ApiError } from './api.error';
+
+export class AmendmentNotCreatedError extends ApiError {
+  constructor(facilityId?: string) {
+    const message = facilityId ? `Amendment not created for facility: ${facilityId}` : 'Amendment not created';
```

Comment:
This is a good touch to the exceptional handling, few comments:

* My understanding is `facilityId` should always exists for an amendment to be processed, can it be please made mandatory.
* Can we also specify why it failed, i.e. unable to get amendment ID, API call and etc.